app/src/main/java/com/infomaniak/mail/ui/main/menuDrawer/MenuDrawerAdapter.kt

```diff
@@ -243,59 +246,59 @@ class MenuDrawerAdapter @Inject constructor() : ListAdapter<Any, MenuDrawerViewH
     abstract class MenuDrawerViewHolder(val binding: ViewBinding) : ViewHolder(binding.root)
 
     private enum class ItemType {
-        CUSTOM_FOLDERS_HEADER,
-        EMPTY_CUSTOM_FOLDERS,
-        DIVIDER,
+        DIVIDER_ITEM,
+        FOLDERS_HEADER_ITEM,
+        EMPTY_CUSTOM_FOLDERS_ITEM,
```

Comment:
You've left the term "custom" in the enum, was it the one you kept removing from the methods in the other files?